src/GeneralStateTestsFiller/stCreateTest/CreateOOGafterMaxCodesizeFiller.yml

```diff
@@ -0,0 +1,347 @@
+    - 0x100000000
+    gasPrice: '10'
+    nonce: '0'
+    to: 00000000000000000000000000000000000c0dea
+    secretKey: "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
+    value:
+    - 0
+
+  expect:
+
+    - indexes:
+        data: !!int 0
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - '>=London'
+      result:
+
+        00000000000000000000000000000000000c0dea:
+          nonce: 1
+          storage:
+            0x1: 0x1
+
+        00000000000000000000000000000000000c0deb:
+          nonce: 1
+          storage: {}
+
+        # 00000000000000000000000000000000000c0deb contracts:
+        # 1
+        76e2310dc9277b22ffceb40d49229131674ebe02:
+          shouldnotexist: 1
+        # 10
+        b3210b741a5dfbddc1636521965b3558defa3e60:
+          shouldnotexist: 1
+
+    - indexes:
+        data: !!int 1
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - '>=London'
+      result:
+
+        00000000000000000000000000000000000c0dea:
+          nonce: 11
+          storage:
+            0x1: 0x1
+
+        00000000000000000000000000000000000c0deb:
+          nonce: 1
+          storage: {}
+
+        # 00000000000000000000000000000000000c0dea contracts:
+        # 1
+        396b47fae89161b1959b5a833acf816be526b860:
+          storage: {}
+        # 10
+        bd205415c24eb289b6a275baae74527b92b49fe2:
+          storage: {}
+        
+        # 00000000000000000000000000000000000c0deb contracts:
+        # 1
+        76e2310dc9277b22ffceb40d49229131674ebe02:
+          shouldnotexist: 1
+        # 10
+        b3210b741a5dfbddc1636521965b3558defa3e60:
+          shouldnotexist: 1
+
+    - indexes:
+        data: !!int 2
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - '>=London'
+      result:
+
+        00000000000000000000000000000000000c0dea:
+          nonce: 11
+          storage:
+            0x1: 0x1
+
+        00000000000000000000000000000000000c0deb:
+          nonce: 11
+          storage:
+            0x1: 0x1
+
+        # 00000000000000000000000000000000000c0dea contracts:
+        # 1
+        396b47fae89161b1959b5a833acf816be526b860:
+          shouldnotexist: 1
+        # 10
+        bd205415c24eb289b6a275baae74527b92b49fe2:
+          shouldnotexist: 1
+        
+        # 00000000000000000000000000000000000c0deb contracts:
+        # 1
+        76e2310dc9277b22ffceb40d49229131674ebe02:
+          shouldnotexist: 1
+        # 244
+        936d5e529580a9cc92bc2b2963a9d0a99f437e89:
+          shouldnotexist: 1
+        # 245
+        2e5a5ec103443f6b299cf31a52c69cf222ae4e6b:
+          storage: {}
+        # 246
+        bade62b355fe6b7117f4f7c913321b318ca3a4da:
+          storage: {}
+        # 247
+        d8a8f3569a1d76027f9ece5010489576897014ea:
+          storage: {}
+        # 248
+        fcdccc3b46b9fbe71221061091e8fe82b77e02f2:
+          storage: {}
+        # 249
+        67262aa10552371ec665c2f90c56d9d65c16715e:
```

Comment:
can put sstore codesize here to verify the size of deployed code

Reply:
Changed the created max codesize contracts to:
- Always do sstore(0, codesize()), so we get the codesize in storage key 0, and verify their size
- Self-destruct if there is calldata (any value)

And we can now call the contracts twice, first to sstore the codesize, and second to self-destruct if necessary.